mods/cnc/chrome/settings.yaml

```diff
@@ -508,43 +515,43 @@ Container@SETTINGS_PANEL:
 							Height: 20
 							Font: Regular
 							Text: Fetch Community News
-						Checkbox@PERFGRAPH_CHECKBOX:
+						Checkbox@CENSUS_CHECKBOX:
 							X: 310
 							Y: 70
 							Width: 300
 							Height: 20
 							Font: Regular
-							Text: Show Performance Graph
+							Text: Participate in Player Census
```

Comment:
Could we be a bit more straightforward and call this (and other necessary places)`Send System Information` instead?